Update landing page, navbar is created

pauljaijai opened the pull request:
I updated the navbar and demo content

pauljaijai committed:
Update landing page, navbar is created

I updated the navbar, and demo contecnt

abc-1211 commented on src/components/Lpcontent/Lpcontent.js:
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './Lpcontent.css';
```
CSS file should be imported to the same file as the class, in this case the file should be imported to Topservicescard.js

abc-1211 commented on src/components/Lpcontent/Lpcontent.css:
```diff
@@ -0,0 +1,13 @@
+.top-service-card{
```
CSS file name should be changed to Topservicecard.css as the css file is styling the component Topserviceard

abc-1211 commented on src/components/Lpcontent/Topservicecard.js:
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+
+function Topserviceard(props){
```
Spelling mistake, please review whether is a typo or its intended.
other than that, as i mentioned, every new word should be capitalize, in this case, it will be
TopServiceCard

abc-1211 commented on src/components/Lpcontent/Topservicecard.js:
```diff
@@ -0,0 +1,21 @@
+
+function Topserviceard(props){
+    return(
+        
```
Extra empty space is not needed, please review

abc-1211 commented on src/components/Lpcontent/Lpcontent.js:
```diff
@@ -0,0 +1,27 @@
+            
+            
+            </div>
+            )    
```
styling please stick to the same format:
```
function Example(props) {
   return (
       <React.Fragment>
             ...content
       </React.Fragment>
   )
}
```

abc-1211 replied:
Noted the open and closing brackets should have the same distance to the left hand side.
E.g. the closing bracket of return should be vertical align with the word return

abc-1211 commented on src/components/navbar/navbar.css:
```diff
@@ -4,4 +4,35 @@ nav {
 .defineCenter {
     float: none !important;
+}
+
```
Alert that searchBtn ("Go!" Button) when hover, the cursor is not turned into pointer, this should be added and amended.
![image](https://user-images.githubusercontent.com/59509722/105933488-74b90280-6089-11eb-9084-dfa50bce81ea.png)

abc-1211 replied:
SearchBox have a line which should not be there
![image](https://user-images.githubusercontent.com/59509722/105933470-6cf95e00-6089-11eb-9deb-5c0d3b56adfe.png)

abc-1211 commented on src/pages/landingpage.js:
```diff
@@ -2,12 +2,14 @@ import React from "react";
     return (
         <React.Fragment>
             <Navbar/>
+            <Lpcontent/>
```
code inside Lpcontent should be moved inside LandingPage.js. Lpcontent is not needed

abc-1211 reviewed (changes requested):
Other Question:
![image](https://user-images.githubusercontent.com/59509722/105934302-e6458080-608a-11eb-9dce-dbf7400b65c7.png)
![image](https://user-images.githubusercontent.com/59509722/105934309-e9407100-608a-11eb-8938-f25aadefb2ba.png)
![image](https://user-images.githubusercontent.com/59509722/105934323-ee052500-608a-11eb-8825-33ba3abe7150.png)
![image](https://user-images.githubusercontent.com/59509722/105934333-f3fb0600-608a-11eb-8043-8b5fd7872255.png)
Content is not responsive on mobile and other location
Please review changes and redo the pull request

abc-1211 commented:
Images should be located in a separate folder instead of inside the one of the component folder

pauljaijai committed:
Update landing page top service

The photos of top service were updated

pauljaijai committed:
Updated topservice list UI

I updated the ui with some stupid code. The titles are localed at bottom og the images. Scale up hover effect is deployed
